kibom/component.py

```diff
@@ -330,24 +330,26 @@ def isFitted(self):
 
         # Variants logic (comma-separated list)
         opts = [s.strip() for s in check.split(",")]
-
-        # Negative decisions
-        for opt in opts:
-            # Any option containing a DNF is not fitted
-            if opt in DNF:
-                return False
```

Comment:
What about this check?

Reply:
Another good catch! I got confused by check above which does almost the same, but that's for space-separated options...